.github/workflows/dart.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - uses: cedx/setup-dart@v2
         with:
           release-channel: dev
+      - run: dart --version
```

Comment:
Should we try upstreaming this so it happens by default during setup? (no need to block this on that, but it might be a good change?)

Reply:
Agreed!

I think we'll do this in the actions that @mit-mit is working on.

Until then...